mentoring/public/js/mentoring_assessment_view.js

```diff
@@ -294,11 +298,25 @@ function MentoringAssessmentView(runtime, element, mentoring) {
         if (submitXHR) {
             submitXHR.abort();
         }
-        submitXHR = $.post(handlerUrl, JSON.stringify(data)).success(callback);
+        submitXHR = $.ajax({
+            'type': 'POST',
+            'url': handlerUrl,
+            'data': JSON.stringify(data),
+            'complete': complete_callback,
+            'success' : success_callback
+        });
     }
 
     function submit() {
-        calculate_results('submit', handleSubmitResults)
+        submitDOM.attr('disabled','disabled');
```

Comment:
I left ``submitDOM.attr('disabled','disabled');`` because it is used everywhere in the file, so we'll have two ways of doing the same thing in a single file.